### PR TITLE
Feature/sergeyro/export storybook components

### PR DESCRIPTION
**Pull request by SergeyRoyt**
https://monday.monday.com/boards/2861766393/pulses/3469455321

Webpack doesn't bundle pictures for storybook components (ComingSoon, ComponentName) - will be fixed in another PR

### CODE REVIEW

**Review by niksa-monday — APPROVED** [review]

**Comment by hadasfa** [comment]
Let's not merge it yet. I think we can do this exact change with much less changes. I am checking this right now and we will talk about it soon?

File: README.md
Comment (hadasfa) [inline README.md:42]: Why? I don't think it's correct

File: package.json
Comment (hadasfa) [inline package.json:14]: let's change it to "/storybookUtils/components" because we will probably will export another services and the storybook addon  (if we will change after merge it will be breaking change)

File: package.json
Comment (hadasfa) [inline package.json:28]: why?

File: src/components/AvatarGroup/__stories__/AvatarGroup.stories.mdx
Comment (hadasfa) [inline src/components/AvatarGroup/__stories__/AvatarGroup.stories.mdx:3]: It's looks like that we have some of the files with camel case naming and some with snake case (this is the right name im not sure) so I am now that imports looks a bit weird

File: src/hooks/useAfterFirstRender/__stories__/useAfterFirstRender.stories.mdx
Comment (hadasfa) [inline src/hooks/useAfterFirstRender/__stories__/useAfterFirstRender.stories.mdx:3]: why? we add it to the configurations of storybook so you are not need to import it directly

File: src/hooks/useClickOutside/__stories__/useClickOutside.stories.mdx
Comment (hadasfa) [inline src/hooks/useClickOutside/__stories__/useClickOutside.stories.mdx:4]: same, It's not needed import

File: src/storybook/components/colors-description/colors-description.jsx
Comment (hadasfa) [inline src/storybook/components/colors-description/colors-description.jsx:29]: not sure we want to to export this one

File: src/storybook/components/doc-footer/doc-footer.jsx
Comment (hadasfa) [inline src/storybook/components/doc-footer/doc-footer.jsx:23]: I am thinking we will not export this one. it's only a link to our form

File: src/storybook/components/multiple-story-elements-wrapper/multiple-story-elements-wrapper.jsx
Comment (hadasfa) [inline src/storybook/components/multiple-story-elements-wrapper/multiple-story-elements-wrapper.jsx:13]: same here, it's a hack which I think that we do not want to export right now

File: src/storybook/components/text-colors/text-colors.jsx
Comment (hadasfa) [inline src/storybook/components/text-colors/text-colors.jsx:20]: Maybe we will want to exprot the colors descriptions for other storybook colors docs but I think there is no need to export our table of text colors

File: src/storybook/components/title/title.scss
Comment (hadasfa) [inline src/storybook/components/title/title.scss:5]: why deleting this?

File: src/storybook/decorators/memory-stats.jsx
Comment (hadasfa) [inline src/storybook/decorators/memory-stats.jsx:16]: I am not sure what is the purpose of this component and because of it not sure if we want to export it

File: src/storybook/decorators/vertical-stories.jsx
Comment (hadasfa) [inline src/storybook/decorators/vertical-stories.jsx:18]: I think we do not want to export this... but not sure

File: webpack.config.js
Comment (hadasfa) [inline webpack.config.js:121]: let's remove the file loader added dependency if right now it's not working and in use?

File: src/storybook/components/colors-description/colors-description.jsx
Comment (hadasfa) [reply src/storybook/components/colors-description/colors-description.jsx:29]: Maybe we will want to export a component which allow to document colors but this one depend on our colorsHashMap which will not benefit other storybook which want to document their custom colors.

**Comment by hadasfa** [comment]
Ok I finish reviewing this. Most important notes in my opinion:
1. let's change the entry name because it will contain more things in the future (services and an addon) - like I wrote in my first comment.
2. We want to export only some of the components. There are component which only relevant to us and we do not want to share with other storybook (for example text colors which contains the docs for our text variable and it's not relevant to other storybooks.) Let's separate between the index file we use for our storybook components and components we choose to share.

Less important notes but let's talk about it:
1. Why changing the default port of the storybook?
2. why adding imports for storybook components which already config with automatic import by storybook?

**Review by hadasfa — CHANGES_REQUESTED** [review]
It's a good PR but please read my notes.

File: README.md
Comment (SergeyRoyt) [reply README.md:42]: Just so it be more convenient - monday-ui-components also runs on localhost:7007 and when u r running them together u need to make more actions like 'press y, so it would run on a different port'...
Decided that it's better to change it here

File: package.json
Comment (SergeyRoyt) [reply package.json:14]: Not only components there, also functions and decorators. "/storybookUtils" then?
But maybe we want to make it all on one entry somehow? All of this will be needed all together, I guess

File: package.json
Comment (SergeyRoyt) [reply package.json:28]: Just so it be more convenient - monday-ui-components also runs on localhost:7007 and when u r running them together u need to make more actions like 'press y, so it would run on a different port'...
Decided that it's better to change it here

File: src/storybook/decorators/memory-stats.jsx
Comment (SergeyRoyt) [reply src/storybook/decorators/memory-stats.jsx:16]: This is definitely needed, as it is implementing functionality of this 
image
image

File: src/storybook/decorators/vertical-stories.jsx
Comment (SergeyRoyt) [reply src/storybook/decorators/vertical-stories.jsx:18]: Removed from `published-storybook-components`

File: src/storybook/components/doc-footer/doc-footer.jsx
Comment (SergeyRoyt) [reply src/storybook/components/doc-footer/doc-footer.jsx:23]: Removed from `published-storybook-components`

File: src/storybook/components/multiple-story-elements-wrapper/multiple-story-elements-wrapper.jsx
Comment (SergeyRoyt) [reply src/storybook/components/multiple-story-elements-wrapper/multiple-story-elements-wrapper.jsx:13]: Removed from `published-storybook-components`

File: src/storybook/components/text-colors/text-colors.jsx
Comment (SergeyRoyt) [reply src/storybook/components/text-colors/text-colors.jsx:20]: Removed from `published-storybook-components`

File: src/storybook/components/colors-description/colors-description.jsx
Comment (SergeyRoyt) [reply src/storybook/components/colors-description/colors-description.jsx:29]: Removed from `published-storybook-components`

File: src/hooks/useAfterFirstRender/__stories__/useAfterFirstRender.stories.mdx
Comment (SergeyRoyt) [reply src/hooks/useAfterFirstRender/__stories__/useAfterFirstRender.stories.mdx:3]: Don't know why, just some sense of perfectionism maybe... :D
Don't want to remove them rn - too many files, but I won't add them from now on, ok?

File: webpack.config.js
Comment (SergeyRoyt) [reply webpack.config.js:121]: Dependency removed 👍

File: package.json
Comment (hadasfa) [reply package.json:14]: we decide to replace it with storybookComponents

**Review by hadasfa — APPROVED** [review]
As we discuss, before merge: undo 7008 + change entry + remove decorators and functions for now

**Pull request merged**
